doc/sphinx/mgmt_preview_quickstart.rst

```diff
@@ -0,0 +1,152 @@
+- **Subscription ID**
+- **Client ID** 
+- **Client Secret** 
+- **Tenant ID** 
+
+These values can be obtained from the portal, here's the instructions:
+
+Get Subscription ID
+^^^^^^^^^^^^^^^^^^^
+
+1. Login into your Azure account
+2. Select Subscriptions in the left sidebar
+3. Select whichever subscription is needed
+4. Click on Overview
+5. Copy the Subscription ID
+
+Get Client ID / Client Secret / Tenant ID 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For information on how to get Client ID, Client Secret, and Tenant ID, please refer to `this document <https://docs.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal>`__
+
+Setting Environment Variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After you obtained the values, you need to set the following values as
+your environment variables
+
+-  ``AZURE_CLIENT_ID``
+-  ``AZURE_CLIENT_SECRET``
+-  ``AZURE_TENANT_ID``
+-  ``AZURE_SUBSCRIPTION_ID``
+
+To set the following environment variables on your development system:
+
+Windows (Note: Administrator access is required)
+
+1. Open the Control Panel
+2. Click System Security, then System
+3. Click Advanced system settings on the left
+4. Inside the System Properties window, click the Environment Variables… button.
+5. Click on the property you would like to change, then click the Edit… button. If the property name is not listed, then click the New… button.
+
+Linux-based OS
+::
+
+    export AZURE_CLIENT_ID="__CLIENT_ID__"
+    export AZURE_CLIENT_SECRET="__CLIENT_SECRET__"
+    export AZURE_TENANT_ID="__TENANT_ID__"
+    export AZURE_SUBSCRIPTION_ID="__SUBSCRIPTION_ID__"
+
+Authentication and Creating Resource Management Client
+------------------------------------------------------
+
+Now that the environment is setup, all you need to do is to create an
+authenticated client. Our default option is to use
+**DefaultAzureCredential** and in this guide we have picked
+**Resources** as our target service, but you can set it up similarly for any other service that you are using.
+
+To authenticate to Azure and create
+a management client, simply do the following:
+
+::
+
+    import azure.mgmt.resource
+    import azure.mgmt.network
+    import azure.mgmt.compute
+    from azure.identity import DefaultAzureCredential
+    ...
+    subscription_id = os.environ.get("AZURE_SUBSCRIPTION_ID")
+    credential = DefaultAzureCredential()
+    resource_client = azure.mgmt.resource.ResourceManagementClient(credential=credential, subscription_id=subscription_id)
+
+More information and different authentication approaches using Azure Identity can be found in
+`this document <https://docs.microsoft.com/python/api/overview/azure/identity-readme?view=azure-python>`__
+
+Managing Resources
+------------------
+
+Now that we are authenticated, we can use our management client to make API
+calls. Let's create a resource group and demonstrate management client's usage
+
+**Create a resource group**
+
+::
+
+    location = "mylocation"
```

Comment:
i would change "mylocation" to some real location for instance "westus2"
if somebody is novice, they might try to put for instance "Warsaw" ;-)

Reply:
yep, good point